### PR TITLE
Update copy/duplicate icon

### PR DESCRIPTION
**Pull request by wfortin**

### CODE REVIEW

**Comment by GermainBergeron** [comment]
:shipit:

File: resources/icons/svg/copy.svg [hunk 1]
```diff
@@ -1,77 +1,35 @@
...
-</g>
-<g>
-	<rect x="128" y="128" width="384" height="384"/>
-</g>
-<polygon points="383.875,0 0.125,0 0.125,384 96.125,384 96.125,96 383.875,96 "/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="20px"
+	 height="20px" viewBox="-284 411.9 20 20" enable-background="new -284 411.9 20 20" xml:space="preserve">
```
Comment (pochretien) [inline resources/icons/svg/copy.svg]: the viewbox continue to be at a weird place ( I know we cant fix that with a editor )

File: resources/icons/svg/copy.svg [hunk 2]
```diff
@@ -1,77 +1,35 @@
...
+<g id="Essai_01" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns" display="none">
+	<title>Icône duplicate</title>
```
Comment (pochretien) [inline resources/icons/svg/copy.svg]: I think we can remove title and desc tag

**Comment by pochretien** [comment]
:ship: :it:

**Pull request merged**